src/Couchbase.TravelSample/Program.cs

```diff
@@ -1,33 +1,80 @@
 using Couchbase.TravelSample.Models;
+using Microsoft.OpenApi.Models;
+using FluentValidation;
+
+using Route = Couchbase.TravelSample.Models.Route;
 
 var builder = WebApplication.CreateBuilder(args);
+const string devSpecificOriginsName = "_devAllowSpecificOrigins";
+
+//global pointer to inventory scope
+IScope? inventoryScope = null;
 
-/// <summary>
-/// dev origins used to fix CORS for local dev/qa debugging of site
-/// </summary>
-const string _devSpecificOriginsName = "_devAllowSpecificOrigins";
+const string airportCollection = "airport";
+const string airlineCollection = "airline";
+const string routeCollection = "route";
 
 // Add services to the container.
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
+builder.Services.AddSwaggerGen(options =>
+{
+    options.SwaggerDoc("v1", new OpenApiInfo
+    {
+        Version = "1.0",
+        Title = "Quickstart in Couchbase with C# and ASP.NET Minimal API",
+        Description = "A quickstart API using C# and ASP.NET with Couchbase and travel-sample data. \n\n"
+                      + "We have a visual representation of the API documentation using Swagger which allows you to interact with the API's endpoints directly through the browser. It provides a clear view of the API including endpoints, HTTP methods, request parameters, and response objects.\n\n"
+                      + "Click on an individual endpoint to expand it and see detailed information. This includes the endpoint's description, possible response status codes, and the request parameters it accepts.\n\n"
+                      + "Trying Out the API\n\n"
+                      + "You can try out an API by clicking on the \"Try it out\" button next to the endpoints.\n\n"
+                      + "- Parameters: If an endpoint requires parameters, Swagger UI provides input boxes for you to fill in. This could include path parameters, query strings, headers, or the body of a POST/PUT request.\n\n"
+                      + "- Execution: Once you've inputted all the necessary parameters, you can click the \"Execute\" button to make a live API call. Swagger UI will send the request to the API and display the response directly in the documentation. This includes the response code, response headers, and response body.\n\n"
+                      + "Models\n\n"
+                      + "Swagger documents the structure of request and response bodies using models. These models define the expected data structure using JSON schema and are extremely helpful in understanding what data to send and expect.\n\n"
+                      + "For details on the API, please check the tutorial on the Couchbase Developer Portal: https://developer.couchbase.com/tutorial-quickstart-csharp-aspnet"
+    });
+});
+
+builder.Services.AddValidatorsFromAssemblyContaining(typeof(AirportCreateRequestCommandValidator));
```

Comment:
You have one assembly why are you adding a single assembly three times?  Do you know what this line of code does?

Reply:
Ok got it. I'll just add it one time as all of these are same type. Done!